scripts/release/version-packages.ts

```diff
@@ -23,8 +24,30 @@ async function versionPackages() {
     await execa('pnpm', ['changeset', 'pre', 'enter', 'canary'], {
       stdio: 'inherit',
     })
-    // TODO: Create empty changeset for `next` to bump canary version
-    // even if there is no changeset.
+
+    console.log(
+      '▲   Preparing to bump the canary version, checking if there are any changesets.'
+    )
+    // Create an empty changeset for `next` to bump the canary version
+    // even if there are no changesets for `next`.
+    const res = await execa('pnpm', ['changeset', 'status'], {
+      // If there are no changesets, this will error. Set reject: false
+      // to avoid trycatch and handle the rest based on the stdout.
+      reject: false,
+    })
+
+    console.log('▲   Changeset Status:')
+    console.log({ ...res })
+
+    if (!res.stdout.includes('- next')) {
+      console.log(
+        '▲   No changesets found for `next`, creating an empty changeset.'
+      )
+      await writeFile(
+        join(process.cwd(), '.changeset', `next-canary-${Date.now()}.md`),
+        `---\n'next': patch\n---`
+      )
```

Comment:
Since this is just temporary until we hard require a changeset, would be good to add a TODO to remove this in the future so we don't publish empty releases.